### PR TITLE
Add metadata to the data_models.py in plotting example

### PR DESCRIPTION
**Pull request by AkintolaRichard**
- [x] issues: fixes #11765
- [x] tests added / passed

### CODE REVIEW

File: examples/plotting/data_models.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
+'''This example shows creating sinusoidal wave using bokeh DataModel.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.model.DataModel, bokeh.models.ColumnDataSource, bokeh.models.CustomJS, bokeh.core.properties.Float
```
Comment (bryevdv) [inline examples/plotting/data_models.py]: typo and suggestion:
```suggestion
    :apis: bokeh.models.DataModel, bokeh.models.ColumnDataSource, bokeh.models.CustomJS
```

File: examples/plotting/data_models.py [hunk 2]
```diff
@@ -1,3 +1,10 @@
+'''This example shows creating sinusoidal wave using bokeh DataModel.
```
Comment (bryevdv) [inline examples/plotting/data_models.py]: nit: consistency with docstring format everywhere else in the repo
```suggestion
''' This example shows creating sinusoidal wave using bokeh DataModel.
```

**Review by bryevdv — COMMENTED** [review]
Thanks @AkintolaRichard sorry for the delay, just two small comments

**Comment by AkintolaRichard** [comment]
It is no problem @bryevdv , I believe you must have been quite busy. I will make the changes as suggested.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @AkintolaRichard

**Pull request merged**